int64 chunk_x
int64 chunk_z
bool continuous
uint16 primary_bitmap
int8[] data
